### PR TITLE
[#34]  NEW DESIGN VERSION Implement user profile web page 

### PR DESCRIPTION
**Pull request by GetFire**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/UserPageVer1.2.html [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+                                            </div>
+                                            <div class="col-xs-6">
+                                                <div class="right-box">Still needed</div>
+                                                <div class="right-box"><span class="fa fa-dollar"></span>9999</div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <a href="#"><img class="card-img-top img-responsive"
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/UserPageVer1.2.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/webapp/WEB-INF/views/UserPageVer1.2.html [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+                    <h1 class="cta-title">Add new Event</h1>
+                    <div class="cta-desc">
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-7">
+                    <form class="form-horizontal" action="" method="" id="addEvent">
+                        <fieldset>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/UserPageVer1.2.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/webapp/WEB-INF/views/UserPageVer1.2.html [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+                    <h1 class="cta-title">Add new Project</h1>
+                    <div class="cta-desc">
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-7">
+                    <form class="form-horizontal" action="" method="" id="addProject">
+                        <fieldset>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/UserPageVer1.2.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by Potyomkin-Igor** [comment]
Классно сделал!!! исправь, так как мы все обговорили

File: src/main/webapp/WEB-INF/views/UserPage.html [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+                                            </div>
+                                            <div class="col-xs-6">
+                                                <div class="right-box">Still needed</div>
+                                                <div class="right-box"><span class="fa fa-dollar"></span>9999</div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <a href="#"><img class="card-img-top img-responsive"
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/UserPage.html:195]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/webapp/WEB-INF/views/UserPage.html [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+                    <a href="#"><h1 class="cta-title">Interests</h1></a>
+                    <div class="cta-desc">
+                        <p><a href="#">My Interests</a></p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-3 form-field">
+                    <form class="form-horizontal" action="" method="" id="addInterest">
+                        <fieldset>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/UserPage.html:311]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/webapp/WEB-INF/views/UserPage.html [hunk 6]
```diff
@@ -0,0 +1,376 @@
...
+                    <a href="#"><h1 class="cta-title">Events</h1></a>
+                    <div class="cta-desc">
+                        <p><a href="#">My Events</a></p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-3 form-field">
+                    <form class="form-horizontal" action="" method="" id="addEvent">
+                        <fieldset>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/UserPage.html:343]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/webapp/WEB-INF/views/UserPage.html [hunk 7]
```diff
@@ -0,0 +1,376 @@
...
+                    <a href="#"><h1 class="cta-title">Projects</h1></a>
+                    <div class="cta-desc">
+                        <p><a href="#">My Projects</a></p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-3 form-field">
+                    <form class="form-horizontal" action="" method="" id="addProject">
+                        <fieldset>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/UserPage.html:283]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 4 issues
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Review by zozich — APPROVED** [review]
Good :)

**Pull request merged**